README.md

```diff
@@ -173,13 +187,28 @@ python train_reader.py \
 	--output_dir {path to output dir}
 ```
 
-Notes: 
+Notes:
 - if you use pytext_bert or fairseq_roberta, you need to download pre-trained weights and specify --pretrained_file parameter. Specify the dir location of the downloaded files for 'pretrained.fairseq.roberta-base' resource prefix for RoBERTa model or the file path for pytext BERT (resource name 'pretrained.pytext.bert-base.model').
-- Reader training pipeline does model validation every --eval_step batches 
+- Reader training pipeline does model validation every --eval_step batches
 - As the bi-encoder, it saves model checkpoints on every validation
 - Like the bi-encoder, there is no stop condition besides a specified amount of epochs to train.
 - Like the bi-encoder, there is no best checkpoint selection logic, so one needs to select that based on dev set validation performance which is logged in the train process output.
-- Our current code only calculates the Exact Match metric. 
+- Our current code only calculates the Exact Match metric.
+
+## Reader model inference
+
+In order to make an inference, run `train_reader.py` without specifying `train_file`. Make sure to specify `model_file` with the path to the checkpoint, `passages_per_question_predict` with number of passages per question (being used when saving the prediction file), and `eval_top_docs` with a list of numbers for number of passages per question which results to be printed. The example command line is as follows.
```

Comment:
"with a list of numbers for number of passages per question" - sounds a bit confusing.
May be rephrase it as "with a list of top passages threshold values from which to choose question's answer span"

Reply:
sounds good!